--- dwm.c.bak	2020-05-30 03:12:53.169118250 +0200
+++ dwm.c	2020-05-30 03:13:19.295491902 +0200
@@ -719,7 +719,7 @@
 	Client *c;
 
 	/* draw status first so it can be overdrawn by tags later */
-	if (m == selmon) { /* status is only drawn on selected monitor */
+	if (m == selmon || 1) { /* status is only drawn on selected monitor */
 		drw_setscheme(drw, scheme[SchemeNorm]);
 		tw = TEXTW(stext) - lrpad + 2; /* 2px right padding */
 		drw_text(drw, m->ww - tw, 0, tw, bh, 0, stext, 0);
@@ -2070,6 +2070,7 @@
 updatestatus(void)
 {
 	char text[512];
+	Monitor* m;
 	if (!gettextprop(root, XA_WM_NAME, text, sizeof(text))) {
 		strcpy(stext, "dwm-"VERSION);
 		estext[0] = '\0';
@@ -2083,7 +2084,8 @@
 		}
 		strncpy(stext, text, sizeof(stext) - 1);
 	}
-	drawbar(selmon);
+	for(m = mons; m; m = m->next)
+		drawbar(m);
 }
 
 void
